aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -196,11 +205,15 @@ module aptos_framework::account {
     /// `new_address`.
     public(friend) fun create_account(new_address: address): signer {
         // there cannot be an Account resource under new_addr already.
-        assert!(!exists<Account>(new_address), error::already_exists(EACCOUNT_ALREADY_EXISTS));
+        assert!(
+            !exists<Account>(new_address),
+            error::already_exists(EACCOUNT_ALREADY_EXISTS)
+        );
 
         // NOTE: @core_resources gets created via a `create_account` call, so we do not include it below.
         assert!(
-            new_address != @vm_reserved && new_address != @aptos_framework && new_address != @aptos_token,
+            new_address != @vm_reserved && new_address != @aptos_framework && new_address
+                != @aptos_token,
```

Comment:
This line break does not seem correct.

Reply:
I prefer
```
new_address != @vm_reserved 
    && new_address != @aptos_framework 
    && new_address != @aptos_token,
```

Reply:
fixed at https://github.com/movebit/movefmt/commit/06f2daf949649e1b4f9f6b1fb1df64ab7620a26e